gtsfm/frontend/matcher/twoway_matcher.py

```diff
@@ -50,6 +57,13 @@ def match(
             descriptors_i2: descriptors corr. to keypoints_i2.
             im_shape_i1: shape of image #i1, as height, width.
             im_shape_i2: shape of image #i2, as height, width.
+            ratio: value for Lowe's ratio test.
+            cross_check: will only return pairs (i,j) such that for i-th query descriptor the j-th descriptor in the
+                matcher's collection is the nearest and vice versa, i.e., will only return consistent pairs.
+            with_scale: take scale transformation into account in GMS.
+            with_roation: take rotation transformation into account in GMS.
+            threshold_factor: threshold factor for rejecting grid cells (see section 3.1 of [1]). The higher, the less
+                matches.
```

Comment:
no mention of `use_gms`.

Can you explain in the PR comments on what the scale/rotation/threshold args are, and when do we turn them on?

Reply:
Good catch, I will add `use_gms`.

I refer to the reader to the paper for more information. I think it's relatively obvious that you'd turn on `with_scale` and `with_rotation` if you're expecting any significant scale or rotation changes between images. It's explained more in the paper.

Reply:
Would it make sense to make GMS a child class of TwoWayMatcher?